### DISCUSSION TITLE
posts/2024/01/how-forest-think/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#34] [General]
# posts/2024/01/how-forest-think/

https://guozheng.netlify.app/posts/2024/01/how-forest-think/

**Comment by liechi** [comment]
>今天晚上，很好的月光。  
我不见他，已是三十多年；今天见了，精神分外爽快。才知道以前的三十多年，全是发昏；然而须十分小心。不然，那赵家的狗，何以看我两眼呢？  
我怕得有理。  

鲁迅似乎也超越了”人类“啊：）

Being 和 creature 的一个区别是，being 可以确定自己存在的同一性（即，我是我）以及具备能动性---这种能力一般被认为是人特有的。而 Creature 是异于人的活物。我没有看过你提到的这本书，但从你的介绍来看，根据作者的思想，不用 Creature 而用 Being 是自然的。

> 我读得云里雾里，但至少作者不是在当“神棍”……

如果你处于云里雾里，如何知道作者不是在跳大神呢？或者说，作者可能不是故意要弄玄虚，但如何确认他不是陷入了某种臆想？

**Reply by residualsun1** [reply]
> 鲁迅似乎也超越了”人类“啊：）

“迅哥儿的想法和思想总先于时代一步”又多了一个案例🤔。

> being 可以确定自己存在的同一性（即，我是我）以及具备能动性---这种能力一般被认为是人特有的。而 Creature 是异于人的活物。

这句涨知识了。话说还有human和human being的区别，前者强调人类这一人群，后者指人群中的每个人。creature我第一次记的时候是“生物”的意思，这也和“异于人的活物”释义相符，但要将中文中“生物”可以包含的“人”从中排除。

> 如果你处于云里雾里，如何知道作者不是在跳大神呢？或者说，作者可能不是故意要弄玄虚，但如何确认他不是陷入了某种臆想？

我迷糊的主要是他讲符号学的部分，但感觉他有一些内容能讲得让人稍微眼前一亮。不过相比故弄玄虚，我更觉得他有可能会偏向一点臆想。我现在陷入了对社会科学和人文学科的迷思，相比可标准化、重复和验证的自然科学，文科百家之言，似乎是谁讲得有理便可行，那这理在哪，标准是啥，谁来给？之前聊梁永佳老师用老子研究库拉圈，这似乎很新颖，还在学科行业内得到了认可，可是然后呢……或许这也只是一种“臆想”，但也可能是他们口中的“丰富对这个世界的理解”。
